.github/workflows/build.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         include:
           - id: normal
             name: normal
-            dependencies: texlive-latex-extra texlive-fonts-recommended hevea sass
+            dependencies: texlive-latex-extra texlive-fonts-recommended hevea sass gdb lldb python3-lldb-14
```

Comment:
Is the `-14` necessary? Does `python3-lldb` work? (it would simplify updating when gha moves to a newer Ubuntu)

Reply:
This should become just `gdb lldb` when we start using Ubuntu 24.04.